### PR TITLE
Wislistandcart

### PR DESCRIPTION
**Pull request by sanu29**
image
image

### CODE REVIEW

File: components/product/product.css [hunk 1]
```diff
@@ -1,74 +1,81 @@
-.card{
+.card {
     width: 15rem !important;
     max-width: 20rem;
     min-width: fit-content;
     margin: 0.5rem !important;
```
Comment (dev-prabhat) [inline components/product/product.css:5]: why important

**Review by Omkar-Ghate — APPROVED** [review]
UI Improvement: The cards and the price details container can be centered for a better UI.

Appreciation: The color combination looks great. Overall, the UI looks neat.

**Review by dev-prabhat — APPROVED** [review]
UI is clean, color combination is good and naming of classes is relatable what it is meant to be

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

**Pull request merged**